app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script.rb

```diff
@@ -29,10 +29,14 @@ def raw_delete_in_provider
   end
 
   def run(vars = {})
-    workflow      = ManageIQ::Providers::AnsiblePlaybookWorkflow
-    extra_vars    = merge_extra_vars(vars[:extra_vars])
-    playbook_vars = { :playbook_path => parent.path }
-    credentials   = collect_credentials(vars)
+    workflow = ManageIQ::Providers::AnsiblePlaybookWorkflow
```

Comment:
Just a note:  The space change here makes this look like a much bigger change then it was.

Unsolicited opinion:  I liked the previous version's spacing better 😜

Reply:
I split up the variables into sections to match the order they are passed as parameters, and once I added the newlines, then the extra spacing looked bad.